.github/workflows/cmake.yml

```diff
@@ -23,17 +27,17 @@ jobs:
           submodules: 'recursive'
 
       - name: Setup Dependencies
+        if: startsWith(matrix.os, 'ubuntu')
         run: |
           sudo apt-get update -y -qq
-          sudo apt-get install build-essential ninja-build
+          sudo apt-get install -y build-essential clang-tidy ninja-build
+
+      - name: Setup Dependencies
+        if: startsWith(matrix.os, 'windows')
```

Comment:
Please use MSCV on windows for building, it should also be preinstalled; you can probably copy from here https://github.com/machineware-gmbh/mwr/blob/main/.github/workflows/nightly.yml